shared/fs/common/use-fs-load-effect.tsx

```diff
@@ -28,13 +28,15 @@ const useFsLoadEffect = ({
     [dispatch, path, isPathItem]
   )
 
-  const online = Container.useSelector(state => state.fs.kbfsDaemonStatus.online)
+  const kbfsDaemonConnected =
+    Container.useSelector(state => state.fs.kbfsDaemonStatus.rpcStatus) ===
+    Types.KbfsDaemonRpcStatus.Connected
   const load = React.useCallback(
     (refreshTag?: Types.RefreshTag) => {
-      online && wantPathMetadata && loadPathMetadata(refreshTag)
-      online && wantChildren && loadChildren(refreshTag)
+      kbfsDaemonConnected && wantPathMetadata && loadPathMetadata(refreshTag)
+      kbfsDaemonConnected && wantChildren && loadChildren(refreshTag)
     },
-    [online, wantChildren, wantPathMetadata, loadPathMetadata, loadChildren]
```

Comment:
did you fix this in the other PR that touches this code?

Reply:
Yeah, that PR is actually based after this: https://github.com/keybase/client/pull/18057/files#diff-6ed8e652e7555af697a744d854958a92R47